src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -340,6 +340,8 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
                 } else {
                     this.exercise.studentParticipations = [...this.studentParticipations, changedParticipation];
                 }
+                this.gradedStudentParticipation = this.participationService.getSpecificStudentParticipation(this.studentParticipations, false);
```

Comment:
How about creating two function `participationService.getGradedParticipation(...)`, `participationService.getPracticeParticipation(...)` instead of using a boolean flag?

Reply:
I'd like to keep the changes small in this bugfix PR.